Makefile

```diff
@@ -988,3 +988,6 @@ docker:
 
 # This endif closes the if at the top of the file
 endif
+
+# Disable parallel execution because it would break some targets that don't specify exact dependencies
```

Comment:
Namely?
Can't we fix them instead of globally disabling parallelism?

Reply:
`backend` does not depend on `frontend` because in the past there were people who refused to install Node.js so we enabled them to use prebuild frontend in the `src` tarballs, it's mentioned in the README:

> When building from the official source tarballs which include pre-built frontend files, the frontend target will not be triggered, making it possible to build without Node.js.

TBH, we could drop the prebuilt frontend in the src tarballs, but it might upset some people, but I do think it's the right thing to do.

Reply:
On the other hand, there is not much to parallelize anyways in the gitea build as there aren't many fan-out cases in the step dependency tree.

Reply:
Comment expanded with those details. We can maybe fix that issue later, but it would be a potentially breaking build change as per above.